src/commands/templates.rs

```diff
@@ -570,6 +583,7 @@ fn skipped_reason_text(reason: &SkippedReason) -> String {
     match reason {
         SkippedReason::AlreadyExists => "Already exists".to_owned(),
         SkippedReason::InvalidManifest(msg) => format!("Template load error: {}", msg),
+        SkippedReason::IdNotIncluded => "ID not included".to_owned(),
```

Comment:
I'm a bit torn on this.  Part of me wants to not print it because it wasn't within the scope of what the user asked to install.  But another part of me notes that if the user typoes an ID then it will help them find what it was meant to be.

In any event if we do print it I'd suggest a message such as "Not requested" that reflects user intent (I just want the C template) rather than the implementation (I passed a list of IDs).

Reply:
I think we can have this table instead of the reporter messages with the reason "Not requested" as suggested.